test/voice.spec.js

```diff
@@ -0,0 +1,51 @@
+var Promise = require("bluebird");
+var expect = require("chai").expect;
+var supertest = require("supertest-promised");
+var parseXML = require("xml-parser");
+var find = require("lodash/find");
+var app = require("../app");
+var SurveyResponse = require("../models/SurveyResponse");
+var agent = supertest(app);
+
+describe("GET /voice", function() {
+  beforeEach(function() {
+    return SurveyResponse.remove({});
+  });
+
+  it("Should return TwiML response with correct message.", function() {
+    var requestBody = {
+      "Called":"+17072053552",
+      "ToState":"CA",
+      "CallerCountry":"US",
+      "Direction":"inbound",
+      "CallerState":"TX",
+      "ToZip":"94595",
+      "CallSid":"CAf20aef736dccc79518789334ba8f73f3",
+      "To":"+17072053552",
+      "CallerZip":"",
+      "ToCountry":"US",
+      "ApiVersion":"2010-04-01",
+      "CalledZip":"94595",
+      "CalledCity":"WALNUT CREEK",
+      "CallStatus":"ringing", "From":"+17378742833", "AccountSid":"AC4ee8a4bf66c95837fc46316395718baa",
```

Comment:
**Important**. As a rule of thumb, we never should push real data in our tests. I have the feeling that this one is a REAL account SID. The same criteria applies to the phone number.

The the assumption I made above is true, please make the appropriate changes.